src/bitwarden_exporter/bw_cli.py

```
@@ -42,17 +43,26 @@ def download_file(item_id: str, attachment_id: str, download_location: str) -> N
+        is_raw=False,
+        settings=settings,
+    )
 
 
 def bw_exec(
-    cmd: List[str], ret_encoding: str = "UTF-8", env_vars: Optional[Dict[str, str]] = None, is_raw: bool = True
+    cmd: List[str],
+    settings: BitwardenExportSettings,
+    ret_encoding: str = "UTF-8",
+    env_vars: Optional[Dict[str, str]] = None,
+    is_raw: bool = True,
```

Ruff (0.14.1)

Boolean-typed positional argument in function definition

(FBT001)